chart/kubeapps/templates/apprepository/deployment.yaml

```diff
@@ -76,6 +76,7 @@ spec:
             {{- end }}
             - --repo-sync-cmd=/asset-syncer
             - --namespace={{ .Release.Namespace }}
+            - --global-repos-namespace={{ .Release.Namespace }}-{{ .Values.apprepository.globalReposNamespaceSuffix }}
```

Comment:
Preferred to add a proper flag instead of passing the value through env variables

Reply:
+100.